consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/converter/XmlToObjectTest.java

```diff
@@ -56,4 +59,27 @@ void shouldConvertBreathingSpaceRequestAndConvertToJson() throws IOException {
         assertNotNull(jsonString);
         assertEquals(JSON_RESPONSE, jsonString);
     }
+
+    @Test
+    void shouldConvertClaimStatusUpdateRequestToString() throws IOException {
+        String xmlContent = readXmlAsString(CLAIM_STATUS_UPDATE);
+        String request = xmlToObject.convertXmlToJson(xmlContent);
+        assertNotNull(request);
+    }
+
+    @Test
+    void shouldConvertClaimStatusUpdateRequest() throws IOException {
+        String xmlContent = readXmlAsString(CLAIM_STATUS_UPDATE);
+        ClaimStatusUpdateRequest request = xmlToObject.convertXmlToObject(xmlContent, ClaimStatusUpdateRequest.class);
+        assertNotNull(request);
+    }
+
+    @Test
+    void shouldConvertClaimStatusUpdateRequestAndConvertToJson() throws IOException {
+        String xmlContent = readXmlAsString(CLAIM_STATUS_UPDATE);
+        ClaimStatusUpdateRequest request = xmlToObject.convertXmlToObject(xmlContent, ClaimStatusUpdateRequest.class);
+        ObjectMapper objectMapper = new ObjectMapper();
+        String jsonString = objectMapper.writeValueAsString(request);
+        assertNotNull(jsonString);
```

Comment:
Add code to check that the content of the jsonString is what is expected

Reply:
done